src/test/java/com/google/api/codegen/testdata/discoveries/java/java_cloudbilling.v1.json.baseline

```diff
@@ -232,7 +232,7 @@ public class CloudbillingExample {
 
     // TODO: Change placeholders below to values for parameters to the 'getBillingInfo' method:
     // * The resource name of the project for which billing information is retrieved. For example, `projects/tokyo-rain-123`.
-    String name = "";
```

Comment:
Don't use "MY-PROJECT". It already exists, and if people run this code without changing the project, they will get a permission denied error.